SiteOps/MyRSEs.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+#
+# MyRses.py - find all rses that I have authority to approve
+#
+
+import sys
+import os
+from rucio.client.client import Client
+
+c = Client()
+
+def usage():
+    print "usage:"
+    print
+    print "%s [-h] [<account>]"%(os.path.basename(sys.argv[0]))
+    print
+    print "    -h this message"
+    print "       without <account>, the current account takes place."
+
+def myrses(account=None):
+    rses = []
+    if account == None:
+        account = c.whoami()['account']
+
+    for r in c.list_rses():
+        attributes = c.list_rse_attributes(r['rse'])
+        if (attributes.has_key('quota_approvers') and attributes['quota_approvers'].find(account) != -1) \
```

Comment:
Can do both of these with "in" I think. 
`if 'quota_approvers' in attributes and account in attributes['quota_approvers']`

Reply:
Changed accordingly.